.gitignore

```diff
@@ -27,5 +27,5 @@ docs/resources/_gen/
 # documentation tool artifact
 tsdoc-metadata.json
 
-# PNPM store (when opening local file system in dev container)
+# PNPM store (when mounting host file system in docker container)
 .pnpm-store/
```

Comment:
Pedantically tweaking the comment from a previous fix for Routerlicious (which builds separately from GitRest and Historian) to use the same phrasing.